integration_tests/testsuite/test_custom.py

```diff
@@ -14,22 +14,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import re
 import json
 import logging
+import requests
 import unittest
 import urlparse
 
 import test_util
 
 
 class TestCustom(unittest.TestCase):
+    """This TestCase fetch a configuration from the endpoint '/custom'
+       describing a series of tests, then run each of them and report their
+       results.
+
+    In the case where a test of the series fail, this TestCase will be
+    considered as failed.
+
+    The specification for the custom tests can be found at:
+    https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests#custom-tests # noqa
```

Comment:
`# noqa` ?

Reply:
This avoid to break the url on multiple lines, and still pass the flake8 tests.